### ISSUE TITLE
Ghidra cannot recognize some VFP instructions

### ISSUE THREAD

**Issue by Mivik** [NationalSecurityAgency/ghidra#1758]
**Describe the bug**
Ghidra cannot recognize some VFP instructions such as vmul.f32

**To Reproduce**
here's a sample that multiplies the floats in s22 and s0 then put the result in s22 (armeabi-v7a):
```
00 ba 2b ee     vmul.f32 s22, s22, s0
```
however Ghidra will translate it as:
```
00 ba 2b ee     cdp        p10,0x2,cr11,cr11,cr0,0x0
```

**Environment (please complete the following information):**
 - OS: Ubuntu 19.10
 - Java Version: 11.0.6
 - Ghidra Version: 9.1.2

**Comment by mumbel** [comment]
Yeah, some of those neon instructions look broken.  Looks like those instructions were developed in SLEIGH to only accept as unconditional instructions (cond=15), and you have always (cond=14).

**Comment by Mivik** [comment]
> optimal

Thanks for your explanation! But i couldn't find a way to edit the instruction... If i choose 'Patch instruction', nothing would change after entering the same instruction again (cdp p10,0x2,cr11,cr11,cr0,0x0). And the ByteViewer forbids editing to any instruction.

**Comment by emteere** [comment]
Fixed by #2677
